spec/metanorma/cli/commands/config_spec.rb

```diff
@@ -0,0 +1,70 @@
+RSpec.describe "Config" do
+  before :each do
+    @test_config = Pathname.new(Dir.tmpdir).join(Metanorma::Cli::CONFIG_FILENAME)
+    FileUtils.rm_f(@test_config)
+
+    allow(Metanorma::Cli).to receive(:config_path).and_return(@test_config)
+    expect(Metanorma::Cli).to receive(:config_path)
+  end
+
+  it "get creates config if not exists" do
+    config_cli = Metanorma::Cli::Commands::Config.new
+
+    config_cli.get("arg1")
+
+    expect(File.exist?(@test_config)).to be true
+  end
+
+  it "print whole config" do
+    config_cli = Metanorma::Cli::Commands::Config.new
+
+    config_cli.get
+
+    expect(File.exist?(@test_config)).to be true
+  end
+
+  it "set -> get" do
+    config_cli = Metanorma::Cli::Commands::Config.new
+
+    config_cli.set("cli.agree-to-terms", "true")
+
+    output = capture_stdout { config_cli.get("cli.agree-to-terms") }
+
+    expect(output).to eq("true")
+
+    expect(File.exist?(@test_config)).to be true
+  end
+
+  it "set -> unset -> get" do
+    config_cli = Metanorma::Cli::Commands::Config.new
+
+    config_cli.set("cli.agree-to-terms", "true")
+    config_cli.unset("cli.agree-to-terms")
+
+    output = capture_stdout { config_cli.get("cli.agree-to-terms") }
+
+    expect(output).to eq("nil")
+
+    expect(File.exist?(@test_config)).to be true
+  end
+
+  it "config values have bigger priority then args" do
+    config_cli = Metanorma::Cli::Commands::Config.new
+    config_cli.set("cli.agree-to-terms", "true")
+    config_cli.set("cli.no-install-fonts", "true")
+    config_cli.set("cli.continue-without-fonts", "true")
+
+    result = Metanorma::Cli::Commands::Config.load_configs(
+      {
+        :"agree-to-terms" => false,
+        :"no-install-fonts" => false,
+        :"continue-without-fonts" => false,
+      },
+      [@test_config]
+    )
+
+    expect(result[:"agree-to-terms"]).to be true
+    expect(result[:"no-install-fonts"]).to be true
+    expect(result[:"continue-without-fonts"]).to be true
```

Comment:
For the configuration attributes, what do you think about following ruby's variable naming convention that we way we can avoid lots of processing by ourself,  eg: `agree_to_terms`, `no_install_fonts` and `continue_without_fonts`?

Reply:
I agree

Reply:
@abunashir @opoudjis agree to, please review once you will have a chance:
 - https://github.com/metanorma/metanorma-cli/pull/220
 - https://github.com/metanorma/metanorma/pull/161